argo-workflows/nautobot-interface-sync/deps/nautobot-token-secret.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+data:
+  token: N1pRN1hveURQV1N5OXA1WEVSQlk3THdManpERHZ4d0ZJa3VVVlA2VQo=
```

Comment:
should this be here?

Reply:
i've changed it to made up token, wanna keep it there as an example of structure that we're expecting